xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -305,6 +305,7 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer, public
   bool HasVideo() const override;
   bool HasAudio() const override;
   bool HasRDS() const override;
+  bool HasPVR() const override;
```

Comment:
I am in the process of cleaning out every PVR (and similar) dependency from VP. VP just plays a stream. It does not care if this is PVR or any other source.

maybe CanPause is what you are looking for?